api/src/main/java/javax/enterprise/inject/spi/builder/InjectionPointConfigurator.java

```diff
@@ -41,51 +37,6 @@
 public interface InjectionPointConfigurator {
 
     /**
-     * Read the InjectionPoint information from the given {@link Field}.
-     * All relevant information is overwritten.
-     *
-     * @param field defining the InjectionPoint.
-     * @return self
-     */
-    InjectionPointConfigurator read(Field field);
```

Comment:
Suppose I have a field

```
@ConfigSetting("boop")
private String boop;
```

I could read the field, and add it as an injectable field.  It wouldn't have a PIP since it doesn't have `@Inject`